src/api/enumeration.js

```diff
@@ -0,0 +1,64 @@
+import config from './util/config';
+import axios_cancelable from './util/axiosCancel';
+import setHeaders from './util/setHeaders';
+import { prettifyCountry, prettifyUserType } from './util/prettifiers';
+
+export const getCountries = () => {
+  return axios_cancelable.get(`${config.dataApi}/enumeration/basic/Country`, {
+    headers: setHeaders()
+  }).then(response => {
+    return response.data.map(code => ({
+      code,
+      name: prettifyCountry(code)
+    }));
+  });
+};
+
+export const getContactTypes = () => {
+  return axios_cancelable.get(`${config.dataApi}/enumeration/basic/ContactType`, {
+    headers: setHeaders()
+  }).then(response => {
+    return response.data.map(code => ({
+      code,
+      name: prettifyUserType(code)
+    }));
+  });
+};
+
+export const getLanguages = () => {
+  return axios_cancelable.get(`${config.dataApi}/enumeration/basic/Language`, {
+    headers: setHeaders()
+  }).then(response => response.data);
+};
+
+export const endpointTypes = [
```

Comment:
Are there any reason these are not taken from the API?

Reply:
My bad. Firstly, I found it in the code of the old app and put as it is. I'll double check the API and replace static data with the appropriate request.

Reply:
The registry API has a list of the enumerations it exposes here, it should be all of them:

https://api.gbif.org/v1/enumeration/basic/